### ISSUE TITLE
List users that are local admins on their own machines

### ISSUE THREAD

**Issue by metamoof** [metamoof/lapsang#7]
This probably needs some sort of regularly run report that feeds into a database somewhere. 

We’d need the script to enumerate it, the script to set up the database, and to work out how to make the data source variable in the pbix file so that it doesn’t break if that database doesn’t exist

**Comment by Zardrastra** [comment]
@metamoof Yes it would require a database to record this, there are a few ways to do this, either the devices phone home or something actively queries the environment

**Comment by Zardrastra** [comment]
Using direct access as a VPN solution would be the most reliable way of ensuring that machines are contactable over the internet in a situation where a central monitoring script or GPO would be deployed to monitor the local administrative group of personal devices.

Direct Access connects to a gateway server at the Windows Login screen (other VPN solutions normally activate either after login or through user intervention)

The other benefits towards using DA is that the client machines will not lose trust relationship with the DC as easily (this can be a ticket driver as endpoints would likely need to check in physically to an office).
